recipes/qrupdate/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "qrupdate" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/{{ name }}/{{ name }}/1.2/{{ name }}-{{ version }}.tar.gz
+  sha256: e2a1c711dc8ebc418e21195833814cb2f84b878b90a2774365f0166402308e08
+  patches:
+    # the whole recipe was copied from:
+    # https://github.com/unlhcc/hcc-conda-recipes/tree/master/recipes/qrupdate/1.1.2
+    # with minor modifications
+    - ldflags.patch
+    - osx.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - make
+  host:
+    - libblas
+    - liblapack
+  run:
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libqrupdate.a
+    - test -f ${PREFIX}/lib/libqrupdate.so  # [linux]
+    - test -f ${PREFIX}/lib/libqrupdate.dylib  # [osx]
```

Comment:
You can use `${SHLIB_EXT}` like in https://github.com/conda-forge/libnetcdf-feedstock/blob/614d097e9686fb126bc4f0a055020319dca8ab18/recipe/meta.yaml#L99

Also, do you need the static library? We usually don't ship them or, if they are really required, ship them in a sub-package. The main reason is b/c they are rarely needed but use a lot of space in the package.

Reply:
I have no idea if the static library is needed; I would guess no...

I will simplify using `${SHLIB_EXT}`